synapse/lib/cell.py

```diff
@@ -1322,6 +1322,8 @@ async def fini():
 
         await self._bumpCellVers('cell:storage', (
```

Comment:
If some versions are going to be nexus and some are not ( and they can be applied in interspliced order between the two types ) these feel like they should be different version trackers, no?

Reply:
Maybe? Perhaps another cell name?

Reply:
OBE, removed the use of `_bumpCellVers()`